cli-plugins/apps/command-reference/workload_get.hbs.md

```diff
@@ -28,8 +28,9 @@ At the very end of the command output, a hint to follow up commands is also disp
 ```bash
 tanzu apps workload get rmq-sample-app
 📡 Overview
-   name:   rmq-sample-app
-   type:   web
+   name:        rmq-sample-app
```

Comment:
This comment is actually for line 27 above the bash example, but I couldn't select that line for comment so I'm adding it here...

Please add something like:
*Note: the `Supply Chain` and `Delivery` sections will be conditionally included in the command output depending on whether those resources are present on the target cluster (e.g. if the target includes only build components, the `Delivery` section there would be no `Delivery` resources available).